helper.py

```diff
@@ -157,10 +157,29 @@ def high_res_img(ctx, url, optional_attributes=None):
 
     # Use native srcset attribute for high res images
     markup = ('<img class="{class_name}" src="{url}" '
-              'srcset="{url_high_res} 1.5x"'
-              '{attrs}>').format(url=url, url_high_res=url_high_res,
+              'srcset="{url_high_res} {scale}"'
+              '{attrs}>').format(url=url, url_high_res=url_high_res, scale=scale,
                                  attrs=attrs, class_name=class_name)
 
+    # If we've specified some alternate formats we need to use the <picture> tag instead
+    if len(alt_formats) > 0:
```

Comment:
alt_formats is empty by default so you don't need `len`, `if alt_formats:` should evaluate to False on its own.

Reply:
Ah good point!